src/main/java/RedisSessionManagerT8.java

```diff
@@ -0,0 +1,298 @@
+
+        Hashtable<String, Object> attributes =
+                (Hashtable)SerializationUtils.deserialize(Base64.getDecoder().decode(jedis.get((id + REDIS_ATTRIBUTES_KEY).getBytes())));
+
+        session = getSession(metadata, attributes);
+
+        redisConnectionPool.returnResourceObject(jedis);
+
+        return session;
+    }
+
+    @Override
+    public void remove(Session session) {
+
+        remove(session, false);
+    }
+
+    @Override
+    public void remove(Session session, boolean update) {
+        //Get connection to redis
+        Jedis jedis = redisConnectionPool.getResource();
+
+        //Just remove the attributes not the metadata
+        jedis.del((session.getId() + REDIS_ATTRIBUTES_KEY).getBytes());
+
+        redisConnectionPool.returnResourceObject(jedis);
+    }
+
+    @Override
+    public void processExpires() {
+       //We need to implement the expire process
+    }
+
+    //------------------------------------------------------------------------------------------------------------------
+    // END Sessions
+    //------------------------------------------------------------------------------------------------------------------
+
+
+    //------------------------------------------------------------------------------------------------------------------
+    // START Utils
+    //------------------------------------------------------------------------------------------------------------------
+
+    /**
+     * Get session from metadata and attributes
+     * @param metadata
+     * @param attributes
+     * @return
+     */
+    private StandardSession getSession(Hashtable<String, Object> metadata, Hashtable<String, Object> attributes){
+        StandardSession standardSession = (StandardSession) createEmptySession();
+        standardSession.setValid(Boolean.valueOf((String)metadata.get(METADATA_VALID)));
+        SimpleDateFormat format = new SimpleDateFormat(ISO_DATE_FORMAT);
+        try {
+            standardSession.setCreationTime(format.parse(((String) metadata.get(METADATA_CREATION_TIME))).getTime());
+        } catch (ParseException e) {
+            e.printStackTrace();
+        }
+        standardSession.setMaxInactiveInterval(Integer.valueOf((String) metadata.get(METADATA_MAX_INACTIVE_INTERVAL)));
+        return standardSession;
+    }
+
+
+    /**
+     * Get metadata from a Session
+     * @param session
+     * @return
+     */
+    private Hashtable<String, Object> getMetadata(Session session){
+        Hashtable<String, Object> metadata = new Hashtable<>();
+        metadata.put(METADATA_VALID, String.valueOf(session.isValid()));
+        metadata.put(METADATA_CREATION_TIME, DateFormatUtils.ISO_DATE_FORMAT.format(session.getCreationTime()));
+        metadata.put(METADATA_LAST_ACCESS_TIME, DateFormatUtils.ISO_DATE_FORMAT.format(session.getLastAccessedTime()));
```

Comment:
Why are you using `DateFormatUtils.ISO_DATE_FORMAT` here but the `SimpleDateFormat` instances with a hand-rolled pattern above?